packages/react-components/src/button/src/Button.css

```diff
@@ -169,8 +183,14 @@ a.o-ui-button {
 /* OUTLINE | HOVER */
 .o-ui-button-outline:hover,
 .o-ui-button-outline.o-ui-button-hover {
-    box-shadow: 0 0 0 1px var(--marine-700) inset;
-    color: var(--marine-700);
+    box-shadow: 0 0 0 1px var(--quaternary-border-hover) inset;
```

Comment:
What the hell is "quaternary" lol ?

Reply:
I feel something like "shadow-border-..." might be easier to understand

Reply:
Here the box-shadow is used to create a border, visually it's a border (1px solid) and not a shadow per se, this is why I used the border variable.